core-bundle/contao/templates/forms/form_altcha.html5

```diff
@@ -0,0 +1,26 @@
+<?php $this->extend('form_row'); ?>
+
+<?php
+
+$GLOBALS['TL_CSS'][] = $this->asset('css/altcha.min.css', 'contao-components/altcha');
+$GLOBALS['TL_BODY'][] = sprintf('<script src="%s" type="module"></script>', $this->asset('js/altcha.js', 'contao-components/altcha'));
+
+$this->altchaAttributes->set('workerurl', $this->asset('js/worker.js', 'contao-components/altcha'));
```

Comment:
This does not work because in the magic `__get` of the widget the attributes are generated from scratch everytime.

We should use 

```
<altcha-widget<?= $this->altchaAttributes->set('workerurl', $this->asset('js/worker.js', 'contao-components/altcha')) ?>></altcha-widget>
```

on line 25 instead.

Reply:
Fixed in 0f157cc6fde2ab138138deadbe8fe21188bd0520.